data/locale/en-US.ini

```diff
@@ -109,6 +121,7 @@ NVENC.RateControl.TwoPass="Enable Two Pass"
 NVENC.RateControl.TwoPass.Description="Enable a secondary pass for encoding, which can help with quality and bitrate stability.\nImproves quality slightly at the cost of some GPU time.\nNvidia Turing hardware might actually see a quality degrade from this."
 NVENC.RateControl.Bitrate="Bitrate Limits"
 NVENC.RateControl.Bitrate.Target="Target Bitrate"
+NVENC.RateControl.Bitrate.Target.Description="Target bitrate, in kilobits per second."
```

Comment:
`, in kilobits per second` is part of the general UI, there should be a kbit/s suffix in the field.

Reply:
This leaves me with a tooltip that says "Target bitrate." I'll remove that, too.